Makefile.in

```diff
@@ -46,7 +40,6 @@ clean:
 
 oclean:
```

Comment:
I see most of the omuse things are removed, isn't this also part of that?

Reply:
yea, OMUSE is a completely independent package now, with its own setup.py..(and only depends on amuse-framework)